app/controllers/AddClosureMetadataController.scala

```diff
@@ -25,21 +30,23 @@ class AddClosureMetadataController @Inject()(val controllerComponents: SecurityC
                                              val cache: AsyncCacheApi)
                                             (implicit val ec: ExecutionContext) extends TokenSecurity with I18nSupport {
 
-  def addClosureMetadata(consignmentId: UUID): Action[AnyContent] = standardTypeAction(consignmentId) {
+  def addClosureMetadata(consignmentId: UUID, fileIds: List[UUID]): Action[AnyContent] = standardTypeAction(consignmentId) {
     implicit request: Request[AnyContent] =>
       //  TODO:  Get selectedFileIds from previous page
-    val fileFilters = None
       for {
-        consignment <- consignmentService.getConsignmentFileMetadata(consignmentId, request.token.bearerAccessToken, fileFilters)
+        consignment <- consignmentService.getConsignmentFileMetadata(consignmentId, request.token.bearerAccessToken, Option(FileFilters(None, Option(fileIds), None)))
         defaultFieldForm <- getDefaultFieldsForForm(consignmentId, request)
         updatedFieldsForForm <- {
           cache.set(s"$consignmentId", consignment.consignmentReference, 1.hour)
-          updateFormFields(defaultFieldForm, consignment.files.headOption.map(_.metadata))
+          updateFormFields(defaultFieldForm, consignment.files.headOption.map(_.fileMetadata).getOrElse(Nil))
```

Comment:
Maybe add a comment here to say temporarily updating form with the first file's metadata until solution for multiple file selection.